### PR TITLE
Fixes issue #1873

### LINKED ISSUE #1873: Indeed Jobs: Implement new footer design
We now have an optional `datebadge`property that can be specified for each tile's `item` (should be specified using the `normalize` function.

We need to add a `datebadge` to the Jobs Spice to better display the posting date for each job.

Live examples can be seen in the SeatGeek Spice: https://duckduckgo.com/?q=concerts+in+toronto&ia=concerts

Code example: https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/seat_geek/events_by_city/seat_geek_events_by_city.js#L153

------
IA Page:  http://duck.co/ia/view/indeed_jobs

### PR DESCRIPTION
**Pull request by cinlloc**
Fixes #1873. Should fix both **Indeed Jobs** and **Sublime** design. 

_Please exclude first 2 commits of pull request (Sorry, I'm starting using Git and don't know how to exclude commits)._

### CODE REVIEW

**Comment by daxtheduck** [comment]
@cinlloc Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by moollaza** [comment]
@cinlloc thanks a lot for this!

>Please exclude first 2 commits of pull request (Sorry, I'm starting using Git and don't know how to exclude commits).

It's no problem :) You may want to look into `git cherry-pick` and `git squash` -- neither are necessary though, but if you prefer a cleaner git log, those should help you

File: lib/DDG/Spice/IndeedJobs.pm [hunk 1]
```diff
@@ -18,15 +18,27 @@ attribution github  => ['parker', 'Parker'],
             twitter => ['ourmaninjapan', 'Daniel Davis'],
             web     => ['http://daniemon.com/', 'Daniel Davis'];
 
-triggers any => "///***never trigger***///";
+triggers any => "job", "jobs";
```
Comment (moollaza) [inline lib/DDG/Spice/IndeedJobs.pm]: this should stay as it is. We're using a special internal triggering method. Please see https://github.com/duckduckgo/zeroclickinfo-spice/commit/afa6edb916b3dcb8a9956123e9f4f88ed54fc0f5 for more info

**Comment by moollaza** [comment]
@cinlloc please add "Fixes #1873" to your PR description above. This will ensure that when we merge the PR, the related issues that this fixes will also be closed :+1:

File: lib/DDG/Spice/TranslateOneWord.pm [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package DDG::Spice::TranslateOneWord;
```
Comment (moollaza) [inline lib/DDG/Spice/TranslateOneWord.pm]: Whoops, this shouldn't be in here. 

I think you'll want to move this code to a separate Git branch.

File: share/spice/sublime_packages/footer.handlebars [hunk 3]
```diff
@@ -1,4 +1,11 @@
 <div class="tile__update__info one-line">
-	<i class="ddgsi ddgsi-arrow-down"></i>
-	{{unique_installs}} &middot; Updated {{last_modified}}
+    <div class="gw">
+        <span class="g   opt--left">
```
Comment (moollaza) [inline share/spice/sublime_packages/footer.handlebars]: I think you should be able to forego using the `gw` and `g` classes. Only the `opt--*` classes should be necessary to float these.

File: lib/DDG/Spice/IndeedJobs.pm (hunk 1 above)
Comment (cinlloc) [inline lib/DDG/Spice/IndeedJobs.pm]: As I tried to explain, this shouldn't part of the pull request (I just reverted deep trigger to allow testing with duckpan)

File: lib/DDG/Spice/TranslateOneWord.pm (hunk 2 above)
Comment (cinlloc) [inline lib/DDG/Spice/TranslateOneWord.pm]: Same as above, should definitely not be part of this pull request

**Comment by cinlloc** [comment]
OK, sorry for the mess. I'm not very skilled yet using Git. I'll try to fix this when I can.

**Comment by moollaza** [comment]
> OK, sorry for the mess. I'm not very skilled yet using Git. I'll try to fix this when I can.

No need to apologize! Git can be a little hard to grasp at first. Unfortunately I think the best way to improve at using Git is to use it and learn from mistakes.

If you need more help with getting your branch(es) in order please let us know!

File: lib/DDG/Spice/IndeedJobs.pm (hunk 1 above)
Comment (moollaza) [inline lib/DDG/Spice/IndeedJobs.pm]: Understood :+1:

**Comment by cinlloc** [comment]
Hi. I've just made a proper new pull request with only correct commits: #2251. This pull request #2229 should be closed.
